tendrl/ceph_integration/objects/cluster/__init__.py

```diff
@@ -0,0 +1,22 @@
+from tendrl.commons.etcdobj import EtcdObj
+from tendrl.ceph_integration import objects
+
+
+class Cluster(objects.CephIntegrationBaseObject):
+    def __init__(self, cluster_id=None, name=None,
+                 status=None, *args, **kwargs):
+        super(Cluster, self).__init__(*args, **kwargs)
+
+        self.value = 'clusters/%s'
```

Comment:
I dont feel its correct to have a detail like 
clusters --> cluster_id --> Cluster  --> status

clusters/<fsid> is the actual cluster and status should be attached to this entity.